lib/plugins/milestones.js

```diff
@@ -14,8 +14,8 @@ module.exports = class Milestones extends Diffable {
   }
 
   async find () {
-    const res = await this.github.issues.getMilestones(Object.assign({ state: 'all' }, this.repo))
-    return res.data
+    const options = this.github.issues.listMilestonesForRepo.endpoint.merge(Object.assign({ state: 'all' }, this.repo))
+    return this.github.paginate(options)
```

Comment:
shouldn't this still need to wait for an async response?

Reply:
Good catch. I actually don't think the function needs to be an async function; I can just return the `this.github.paginate()` promise because in [lib/diffable.js](https://github.com/probot/settings/blob/ef2389b786e339471783142e4d391e5a3efd839f/lib/plugins/diffable.js#L31)